shared/css/typography.scss

```diff
@@ -9,7 +13,7 @@
 // identical classes for potential use on other elements if needed.
 
 // ----------------------------------------
-// v 1.0.1
+// v 1.1.1
```

Comment:
Because we've updated typography to use rem recently but forgot to update the version comment